Posts/2021/09/Understanding Get-ACL and AD Drive Output.md

```diff
@@ -0,0 +1,245 @@
+---
+post_title: Understanding Get-ACL and AD Drive Output
+username: farisnt@gmail.com
+Categories: PowerShell
+tags: PowerShell, Active Directory, ACL
+Summary: Understanding Get-ACL and AD Drive Output
+---
+
+## Understanding `Get-ACL` and AD: Drive Output
+
+Understanding Active Directory ACL using PowerShell can be a bit tricky.
+There are no out-of-the-box cmdlets with ActiveDirectory PowerShell module to help in settings the permission quickly.
+While there are no cmdlets, you can nevertheless manage AD permissions using the AD PowerShell drive.
+
+In this post, I will try to simplify Active Directory ACL and how to read the result easily, So let's start.
```

Comment:
In this post, I will try to simplify Active Directory ACL and how to read the result easily, so let's start.

de-capitalise the S in So...